src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1341,13 +1371,20 @@
     <Resource Include="UI\Images\PackageManager\down_16_16.png" />
     <Resource Include="UI\Images\PackageManager\folder_16_16.png" />
     <Resource Include="UI\Images\PackageManager\up_16_16.png" />
+    <Resource Include="UI\Images\pin_hidden_16px.png" />
+    <Resource Include="UI\Images\pin_selected_16px.png" />
+    <Resource Include="UI\Images\Pin_16px.png" />
+    <Resource Include="UI\Images\Watch_16px.png" />
```

Comment:
Can you rename these pictures to be `16_16`?

Reply:
Done.